sunpy_sphinx_theme/conf.py

```diff
@@ -66,23 +57,12 @@ def page_url(page):
             ],
             1,
         ),
-        ("Blog", page_url("blog.html"), 1),
-        ("Support Us", page_url("contribute.html"), 1),
         ("Get Help", page_url("help.html"), 1),
-        (
-            "SunPy Project",
-            [
-                ("SunPy Project", page_url("project/"), 1),
-                ("Community Roles", page_url("project/roles.html"), 1),
-                ("Affiliated Packages", page_url("project/affiliated.html"), 1),
-                ("Emeritus role holders", page_url("project/former.html"), 1),
-                ("Meetings", page_url("project/meetings.html"), 1),
-            ],
-            1,
-        ),
+        ("Support Us", page_url("contribute.html"), 1),
+        ("Blog", page_url("blog.html"), 1),
     ],
-    # Only really setup to look nice with 3 values.
     "footer_links": [
+        ("Code of Conduct", "coc.html", 1),
         ("GitHub", "https://github.com/sunpy/sunpy", 1),
```

Comment:
Also should this link to the org not sunpy?

Reply:
We should change that.